packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -377,6 +387,10 @@ class _RangeSliderState extends State<RangeSlider> with TickerProviderStateMixin
   AnimationController startPositionController;
   AnimationController endPositionController;
   Timer interactionTimer;
+  // Value Indicator paint Animation that appears on the Overlay.
+  Function paintTopValueIndicator;
```

Comment:
We may want to create a type alias for these functions (and the one in slider.dart too)

Reply:
Done